### PR TITLE
Fix newline parsing

### PR DESCRIPTION
**Pull request by penguinuwu**
Remove all newlines caused by HTML code formatting

### CODE REVIEW

File: test/test.ts [hunk 1]
```diff
@@ -1,35 +1,178 @@
...
 // Note: baseUrl is set to src/ https://stackoverflow.com/a/43721170
 
 // Note: test async with chai-as-promised https://stackoverflow.com/a/45496509
 chai.use(chaiAsPromised);
 
 describe("Data parser test", () => {
+  // Test URL parsers
+  describe("Test URL parser", () => {
+    // Positive cases
+    casesDataParserURL.forEach(([desc, url, href, file]) => {
```
Comment (Andi-Fan) [inline test/test.ts:25]: I've actually not seen test cases setup before, where the it() description is also imported. I like it though

File: test/test.ts [hunk 2]
```diff
@@ -1,35 +1,178 @@
...
+        chai.assert.equal(getParsedHref(parsedURL, href), result);
+      });
+    });
+
+    // Negative cases
+    // it("Empty URL", () => {
+    //   const parsedURL = getParsedURL("");
+    //   chai.assert.equal(getParsedHref(parsedURL, "anything"), "");
+    // });
+  });
```
Comment (Andi-Fan) [inline test/test.ts:56]: Does this test not work currently? Not sure if we should be committing commented code

**Review by Andi-Fan — APPROVED** [review]
Looks good, tests seem pretty thorough nice job :D

File: test/test.ts (hunk 2 above)
Comment (penguinuwu) [reply test/test.ts:56]: o oops i forgor 💀

**Pull request merged**